src/views/SemesterDetail.vue

```diff
@@ -0,0 +1,145 @@
+<template>
```

Comment:
This template is largely the same a what I pulled over from the backend project.